ui/src/features/stage/stage-actions.tsx

```diff
@@ -80,20 +86,24 @@ export const StageActions = ({ stage }: { stage: Stage }) => {
   return (
     <Space size={16}>
       {argoCDAppsLinks.length === 1 && (
-        <Button
-          type='link'
-          onClick={() => window.open(argoCDAppsLinks[0]?.url, '_blank', 'noreferrer')}
-          size='small'
-        >
-          Argo CD
-        </Button>
+        <Tooltip title={argoCDAppsLinks[0]?.label}>
```

Comment:
Is the system-wide label we name the instance? If so, should we instead title the link to the name of the application we are linking to?

Reply:
Oh, this is already done, just a few lines above: https://github.com/akuity/kargo/blob/0e8f38e55a98c132643285d84ecb02528c67e427/ui/src/features/stage/stage-actions.tsx#L75